app/controllers/lakeshore/ingest_controller.rb

```diff
@@ -42,6 +41,7 @@ def validate_asset_type
       end
 
       def validate_duplicate_upload
+        return if check_duplicates_turned_off?
```

Comment:
@awead this is all I ever really needed (plus a bunch of tests I wrote) as far as logic goes for respecting the duplicate_check param. I changed the name just so it's a little more specific in what it's returning.